package.xml

```diff
@@ -62,45 +62,74 @@
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
     <notes>
         <![CDATA[
+### Fixes across all extensions
+- Handle Apache graceful restarts more accurately #2483
+- Mark config as immutable and interned to avoid refcounting race conditions #2516
+
 ## Tracer
+This release includes Windows dlls for the tracer starting with PHP 7.2. #2230
+
+Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+
 ### Added
-- Add span links capabilities to the OTel API #2451
-- Add exec integration #2361, #2455, #2459
+- Enhance logging #2490
+  - This features two new logging options:
+    - `datadog.trace.log_level` allows more fine-grained control over logging (`error`, `warn`, `info`, `debug`, `trace`). It follows the RUST_LOG standard. Note that `datadog.trace.debug` still takes precedence if set.
+    - `datadog.trace.log_file` allows specifying a dedicated target file for any datadog trace logs instead of the default php error location.
```

Comment:
Question: Have we updated the documentation?

Reply:
:-)

Yeah, I guess I should PR that now.